components/button/metadata/button-accent.yml

```diff
@@ -258,3 +258,90 @@ examples:
           </button>
         </div>
       </div>
+  - id: button-pending
```

Comment:
A bit torn on how to display this in the docs site. Should it be it's own page? If not, should I show the same pending state examples for all button types since it will look the same for all of them (except maybe static color, waiting to hear back from design on this)?